index.html

```diff
@@ -5061,6 +5061,143 @@ <h2 id="AudioContextInfo">
+            exception and abort all these steps.
+            </li>
+            <li>Let <code><em>processorConstructor</em></code> be the result of
+            looking up name on the <a>AudioWorkletGlobalScope</a>’s <a>node
+            name to processor definition map</a>.
+            </li>
+            <li>Let <code><em>baseNodeInstance</em></code> be a new instance of
+            <a>AudioWorkletNode</a>.
+            </li>
+            <li>Let <code><em>options</em></code> be the optional
+            <code>options</code> argument in the constructor.
+            </li>
+            <li>Let <code><em>processorInstance</em></code> be a new instance
+            of <a>AudioWorkletProcessor</a> created by processorConstructor.
+            </li>
+            <li>Let <code><em>parameterDescriptors</em></code> be the result of
+            <code><a href=
+            "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=processorConstructor,
+            P="parameterDescriptors")</code>. If
+            <code><em>parameterDescriptors</em></code> in
+            <code>undefined</code>, skip to the step 9.
+            </li>
+            <li>For each <code><em>descriptor</em></code> in
+            <code><em>parameterDescriptors</em></code> perform the following
+            substeps:
+              <ol>
+                <li>Let <code><em>audioParam</em></code> be a new
+                <a>AudioParam</a> instance with:
+                  <ul>
+                    <li>
+                      <code><em>paramName</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="name")</code>.
+                    </li>
+                    <li>
+                      <code><em>defaultValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="defaultValue")</code>.
+                    </li>
+                    <li>
+                      <code><em>minValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="minValue")</code>.
+                    </li>
+                    <li>
+                      <code><em>maxValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="maxValue")</code>.
+                    </li>
+                  </ul>
+                </li>
+                <li>Let <code><em>initialValue</em></code> be the result of
+                <code><a href=
+                "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=<em>options</em>,
+                P=<em>paramName</em>)</code>.
```

Comment:
Since options is an argument to your constructor, it will already be processed by Web IDL, and converted to a number (if it is one of the unsigned long members of the dictionary). So these two steps should instead be something like "If options contains a dictionary member with name _paramName_ whose type is unsigned long, perform Set(audioParam, "value", the value of that dictionary member)."